iam-login-service/src/main/java/it/infn/mw/iam/notification/TransientNotificationFactory.java

```diff
@@ -142,6 +145,73 @@ public IamEmailNotification createResetPasswordMessage(IamAccount account) {
     return notification;
   }
 
+  @Override
+  public IamEmailNotification createAdminHandleGroupRequestMessage(IamGroupRequest groupRequest) {
+    String dashboardUrl = String.format("%s/dashboard#/requests", baseUrl);
```

Comment:
this could be set after the component initialization, instead of recreating it for each request